toolchain/driver/driver_test.cpp

```diff
@@ -126,32 +124,8 @@ TEST_F(DriverTest, DumpTokens) {
   EXPECT_TRUE(
       driver_.RunCommand({"compile", "--phase=lex", "--dump-tokens", file}));
   EXPECT_THAT(test_error_stream_.TakeStr(), StrEq(""));
-  auto tokenized_text = test_output_stream_.TakeStr();
-
-  EXPECT_THAT(Yaml::Value::FromText(tokenized_text),
-              ElementsAre(Yaml::SequenceValue{
-                  Yaml::MappingValue{{"index", "0"},
-                                     {"kind", "Identifier"},
-                                     {"line", "1"},
-                                     {"column", "1"},
-                                     {"indent", "1"},
-                                     {"spelling", "Hello"},
-                                     {"identifier", "0"},
-                                     {"has_trailing_space", "true"}},
-                  Yaml::MappingValue{{"index", "1"},
-                                     {"kind", "Identifier"},
-                                     {"line", "1"},
-                                     {"column", "7"},
-                                     {"indent", "1"},
-                                     {"spelling", "World"},
-                                     {"identifier", "1"},
-                                     {"has_trailing_space", "true"}},
-                  Yaml::MappingValue{{"index", "2"},
-                                     {"kind", "EndOfFile"},
-                                     {"line", "1"},
-                                     {"column", "12"},
-                                     {"indent", "1"},
-                                     {"spelling", ""}}}));
+  // Verify there is output without examining it.
+  EXPECT_FALSE(test_output_stream_.TakeStr().empty());
```

Comment:
I'm fine with dropping the test of the contents, but maybe we should pass this through `Yaml::Value::FromText` to make sure it's at least valid YAML?

Reply:
Maybe `Yaml::Value::FromText` should be changed to validate that it's yaml, but it doesn't do that right now, so calling it wouldn't actually have an effect. I'll keep this in mind though for a future change.